tests/operators/test_agnostic_load_file.py

```diff
@@ -534,3 +534,75 @@ def test_load_file(sample_dag, sql_server, file_type):
         ]
     )
     assert df.rename(columns=str.lower).equals(expected)
+
+
+@pytest.mark.parametrize("sql_server", ["bigquery"], indirect=True)
+@mock.patch("pandas.DataFrame.to_gbq")
+def test_load_file_chunks_bq(mock_to_gbq, sample_dag, sql_server):
+    file_type = "csv"
+    sql_name, sql_hook = sql_server
+
+    # While hooks expect specific attributes for connection (e.g. `snowflake_conn_id`)
+    # the load_file operator expects a generic attribute name (`conn_id`)
+    sql_server_params = copy.deepcopy(test_utils.SQL_SERVER_HOOK_PARAMETERS[sql_name])
+    conn_id_value = sql_server_params.pop(
+        test_utils.SQL_SERVER_CONNECTION_KEY[sql_name]
+    )
+    sql_server_params["conn_id"] = conn_id_value
+
+    task_params = {
+        "path": str(CWD) + f"/../data/sample.{file_type}",
+        "file_conn_id": "",
+        "output_table": Table(table_name=OUTPUT_TABLE_NAME, **sql_server_params),
+    }
+    test_utils.create_and_run_task(sample_dag, load_file, (), task_params)
+
+    assert mock_to_gbq.call_args_list[0].kwargs["chunksize"] == 1000000
```

Comment:
I agree with using `constants.py` more often!

In this particular case, I think it is important to be explicit in the test and not use `constants.py`.

The test helps us think if we really should change the chunk size - and it would break f someone set `constants.CHUNKSIZE=1` - which is undesirable.

If the test used the constant CHUNKSIZE, people would be able to change it without even thinking about the consequences - and no tests would break.